common/djangoapps/student/models.py

```diff
@@ -812,7 +812,16 @@ def user_post_save_callback(sender, **kwargs):
 
     changed_fields = user._changed_fields  # lint-amnesty, pylint: disable=protected-access
 
-    if 'is_active' in changed_fields or 'email' in changed_fields:
+    # DISABLE_AUTOENROLL_AFTER_COURSE_END flag is used to disable auto enrollment of user to a course
+    # if user is registering after course has ended
+    disabled_auto_enroll_after_course_end = settings.FEATURES.get('DISABLE_AUTOENROLL_AFTER_COURSE_END', False)
```

Comment:
Nit: this is a default value, so wouldn't `settings.FEATURES['DISABLE_AUTOENROLL_AFTER_COURSE_END']` be better?
This way, a configuration is the single source of truth for a default value.